bottlerocket-settings-models/settings-extensions/kubelet-device-plugins/Cargo.toml

```diff
@@ -10,6 +10,7 @@ publish = false
 bottlerocket-modeled-types.workspace = true
 bottlerocket-model-derive.workspace = true
 bottlerocket-settings-sdk.workspace = true
+bounded-integer = { workspace = true, features = ["types"]}
```

Comment:
This could go in `dev-dependencies`, since it's only used for tests.

Reply:
Updated, thanks for the suggestion.